lib/users.ts

```diff
@@ -9,12 +9,12 @@ import { MongoAbility } from "@casl/ability";
  * Get or Create a user if a record does not exist
  * @returns A User Object
  */
-export const getOrCreateUser = async ({ sub, name, email, picture }: DefaultJWT) => {
-  if (!sub) throw new Error("Sub does not exist on token");
+export const getOrCreateUser = async ({ name, email, picture }: DefaultJWT) => {
+  if (!email) throw new Error("Sub does not exist on token");
```

Comment:
We may want to change the text in the Error thrown as well to `"Email does not exist on token"`